dbfilter_from_header/override.py

```diff
@@ -13,11 +13,13 @@
 
 
 def db_filter(dbs, httprequest=None):
-    dbs = db_filter_org(dbs, httprequest)
+    dbs_orig = db_filter_org(dbs, httprequest)
     httprequest = httprequest or http.request.httprequest
     db_filter_hdr = httprequest.environ.get("HTTP_X_ODOO_DBFILTER")
     if db_filter_hdr:
         dbs = [db for db in dbs if re.match(db_filter_hdr, db)]
+    else:
+        dbs = dbs_orig
```

Comment:
Could you explain what this actually changes, because I don't see it.

```
dbs = something
if condition:
    dbs = something_other
return dbs
```

Should be functionally equivalent to:

```
dbs_orig = something
if condition:
    dbs = something_other
else:
    dbs = dbs_orig
return dbs
```

Reply:
thanks for reviewing
In the previous code, as dbs was already in the arguments, it was overwritten by 
`dbs = something`

Reply:
Ahhh i see it now. Yes